focus-ios/version.xcconfig

```diff
@@ -1 +1 @@
-APP_VERSION = 139.0
+APP_VERSION = 140.0
```

Comment:
Also I commit this change because we did it in Firefox but let me know if we shouldn't